requirements.txt

```diff
@@ -6,7 +6,6 @@ docker==2.7.0
 pexpect==4.3.1
 prettytable==0.7.2
 psutil==5.2.2
-psycopg2==2.7.3.2
```

Comment:
Probably should drop this postgres client requirement in the readme.

Reply:
So annoyingly on Homebrew, you still have to install full `postgresql`, unless you install `libpq` and then link the specific binary as described on [SO](https://stackoverflow.com/a/49689589).

We could describe this process in the readme, or alternatively we could say that we depend on `libpq`, and then use the full path to the `pg_admin` binary in `/usr/local/opt/libpq/bin`.

Thoughts?

Reply:
Agreed 😜 Ideally there would be a pure python library for this; I will keep looking at it.